### ISSUE TITLE
ZeroDivisionError on Sidonia no Kishi

### ISSUE THREAD

**Issue by TotallyNotGovna** [tp7/Sushi#15]
http://www.datafilehost.com/d/6af541dd

Attempting to run sushi produces the error 

ZeroDivisionError: Weights sum to zero, can't be noramlized.

**Comment by tp7** [comment]
Yeah, for some reason Sushi's audio search just doesn't work properly with Sidonia (not only this ep but various others). I'm afraid this issue cannot be fixed easily and I'll have to find some new way to improve search accuracy, which by the looks of it is not happening any time soon.

**Comment by shinchiro** [comment]
In this case, the shift is really 1 second. So, the second check:
<code>abs_diff(new_time - original_time, last_shift) > ALLOWED_ERROR</code> is not necessary. Removing this check will produce correct output without error
